trestle/core/commands/command_docs.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, parser=None, parent=None, name=None, out=None, err=None) -> N
 
     def _validate_arguments(self, args):
         # if the command is 'init' then don't validate the trestle-root as it will be initialized by init command
-        if self.name in ['init', 'trestle', 'version', 'partial-object-validate']:
+        if self.name in ['init', 'trestle', 'version', 'partial-object-validate', 'fedramp-validate']:
```

Comment:
There will be a corresponding issue here. 

My suggestion would be that we separate this into a separate class so classes can decide whether or not they need trestle_root.

Reply:
created a new base class where trestle-root is not checked.